### ISSUE TITLE
Dual SIM on BQ Aquaris E5 not supported

### ISSUE THREAD

**Issue by wrogner** [ubports/ubuntu-touch#899]
### BUG

- Device: BQ Aquaris E5 HD Ubuntu Edition
- Channel: 
- Build: 2018-W40

### Steps to reproduce
Insert SIM into Slot 1
Start device
SIM1 not recognized

### Expected behavior
SIM1 should be recognized
(Before flashing ubuntu-touch I had plain Ubuntu software on it and both SIMs worked fine. I don't think that Slot broke during flashing but have no way to prove it fault free)

### Actual behavior
SIM1 reported empty in System configurations

### Logfiles and additional information
none

**Comment by Flohack74** [comment]
I dont think anything was changed on that low level, but I will update my E5 to test this. Aquaris phones tend to break certain SIM cards over time, I replaced mine every 1.5 months until I figured this out. Which provider have you got?

**Comment by wrogner** [comment]
I did not submit this report lightly.
Unfortunately I cannot revert to the original OS to prove that the SIM slot is still working.

The problem lies within Ubuntu Touch (the SIM did not work from the word go).

The SIM card is not broken, it works fine on a different BQs with Android (in combination with other SIMs). Using this phone with Ubuntu (original SW) two SIMs worked for more than a year without any issues.

The telephone provider is irrelevant as the phenomenon is reproducable with SIMs from different vendors (A1, Hot, Yesss).

**Comment by Gabriele69** [comment]
Hi, 
I've a BQ 4.5 with 2 SIM, without problems.
OS 16.04 Dev

**Comment by Danfro** [comment]
Hi, I have got an BQ E5 for over 3 years now.
Through all versions of ubuntu touch up to 16.04 now I have been using it with two SIM-cards without any issues.
Could it be an hardware issue only with this one single phone?

**Comment by MenmuirMark** [comment]
I've used my BQ E4.5 with dual SIM (personal and work) since I bought it new several years ago. I am still using the original SIM cards (one of which I trimmed to fit with a pair of scissors). Very rarely a SIM card fails to register but reinserting it and restarting the phone has never (yet) failed. I have never had a problem connected to any of the updates affecting the SIM cards. My understanding is that the E4.5 and E5 use the same hardware and Android blobs so perhaps you have just had an unlucky co-incidence. I use EE and Vodaphone in the UK. OS 16.04 Stable
